src/audio/rtnr/rtnr.c

```diff
@@ -549,9 +573,13 @@ static int rtnr_set_config_bytes(struct comp_dev *dev,
 	return ret;
 }
 
-static int rtnr_set_bin_data(struct comp_dev *dev, struct sof_ipc_ctrl_data *cdata)
```

Comment:
@MingJenTai Also need helping hand here. Does this component have two binary controls with SOF_RTNR_CONFIG and SOF_RTNR_DATA. There is no `struct sof_ipc_ctrl_data *cdata` in IPC4 so can't separate them by type. Also the kernel controls cache keeps only the last control instance data. Could this be handled in IPC4 with two bytes controls instances?

Reply:
Yes, there are two binary controls as seen from https://github.com/thesofproject/sof/blob/main/tools/topology/topology1/sof/pipe-rtnr-google-rtc-audio-processing-capture.m4. The current draft PR version now handles the two control. I've converted the supplied blobs for tplg1 to tplg2 now.